src/services/changeAPI/nonProfits.ts

```diff
@@ -0,0 +1,157 @@
+import Axios, { AxiosResponse } from 'axios';
+import slugify from 'slugify';
+import config from '../../config';
+import { Organization, ORGANIZATION_LABELS } from '../../entities/organization';
+import {
+  Category,
+  Project,
+  ProjectUpdate,
+  ProjStatus,
+} from '../../entities/project';
+import { ProjectStatus } from '../../entities/projectStatus';
+import { errorMessages } from '../../utils/errorMessages';
+import { logger } from '../../utils/logger';
+import { getAppropriateSlug, getQualityScore } from '../projectService';
+
+const changeAPICategoryName = 'Change';
+const changeAPIHandle = 'change';
+
+// Admin Account assigned by Giveth to handle this projects
+const adminId =
+  (config.get('THIRD_PARTY_PROJECTS_ADMIN_USER_ID') as string) || '1';
+
+interface ChangeNonProfit {
+  address_line?: string;
+  category?: string;
+  city?: string;
+  classification?: string;
+  crypto: {
+    ethereum_address: string;
+    solana_address?: string;
+  };
+  ein: string;
+  icon_url: string;
+  id: string;
+  mission: string;
+  name: string;
+  socials: {
+    facebook?: string;
+    instagram?: string;
+    twitter?: string;
+    youtube?: string;
+  };
+  state?: string;
+  website?: string;
+  zip_code?: string;
+}
+
+// exact title returns 1 element
+export const getChangeNonProfitByNameOrIEN = async (
```

Comment:
It might be good if you can write test for change APIs, if you want to do that, you have to pass CHANGE API token in github actions like this (https://github.com/Giveth/impact-graph/blob/staging/.github/workflows/CI-CD.yml#L46-L53)
But I don't know if you use a specific project's name, when the project name changes our test will fail, so please do everything you think is fine, I just suggested that

Reply:
Good Suggestion. Ill actually write two tests, one for simulating the API for createProjectFromChangeNonProfit and another for importThirdPartyProject, simulating the error messages.

But we shouldn't add api calls to our test suite in my opinion, as it will slow down our test suite and if their api is down, so is our tests. Calling apis is somewhat slow from what I tested.

Even if the API is down, we log and throw the errors.